WordPress/src/main/java/org/wordpress/android/ui/posts/navigation/EditPostNavigator.kt

```diff
@@ -0,0 +1,27 @@
+package org.wordpress.android.ui.posts.navigation
+
+/**
+ * Interface for handling navigation between different screens in the Edit Post flow.
+ * Provides abstraction over the underlying navigation implementation.
+ */
+interface EditPostNavigator {
```

Comment:
AS says this interface is unused, which I confirmed by deleting it and successfully building the app. Is there a reason to keep this?

Reply:
Addressed in eae928710401ea269ca46e48e1e67480ed9bc90f in #22068.